src/fixtures/slack_action_payload.js

```diff
@@ -0,0 +1,31 @@
+export const makeSlackMessageBlockActionPayload = (props = {}) => ({
```

Comment:
Note that unlike the blocks defined in the block utils, we generate these with factory functions because any mutations will mess with test isolation.